assets/scripts/streets/StreetName.jsx

```diff
@@ -44,10 +45,61 @@ class StreetName extends React.PureComponent {
     super(props)
 
     this.state = {
-      isHovered: false
+      isHovered: false,
+      modify: false,
+      inpVal: ''
     }
   }
 
+  handleStreetNameChange = () => {
```

Comment:
For clarity, place all custom methods _after_ React lifecycle methods.

Although we do not use the AirBnb JSX linter, I have found that their [ordering guidelines](https://github.com/airbnb/javascript/tree/master/react#ordering) to be very helpful. Please use this as a reference when ordering methods within components.